client/src/styles/_modeling.less

```diff
@@ -20,3 +20,10 @@
 .djs-popup-backdrop {
   z-index: 1001;
 }
+
+
+// Patch outdated form-js properties-panel styles
+// TODO: remove once form-js updates the bundled properties-panel styles
+.bio-properties-panel-popup .bio-properties-panel-popup__header .bio-properties-panel-feel-popup__title-link svg {
+  margin-left: 0.5em;
+}
```

Comment:
Since form-js bundles the properties-panel styles again, this change:

https://github.com/bpmn-io/properties-panel/commit/9d5916b343eb7e39c1ae1a896c096cf7ecc5b453#diff-ab3f4300b70b36ecc12a0029960ab730d81c52059b3ef7cde710f147414a8b72L1312-L1315

Clashes with the new styles and would lead to an overlap of the Icon and `learn FEEL` link text

Reply:
I mentioned an approach to consuming the form-js-editor styles in a more controlled manner via https://github.com/camunda/web-modeler/pull/6710#issuecomment-1790843962. 

Maybe something we can consider to make the properties panel styles peer-dependecy'ish for the form editor.

Reply:
Thanks for the hint, it tried it out and it does not load the css twice 😄